app/src/main/java/de/westnordost/streetcomplete/quests/place_name/AddPlaceName.kt

```diff
@@ -113,6 +115,9 @@ class AddPlaceName(
     override fun isApplicableTo(element: Element): Boolean =
         filter.matches(element) && hasFeatureName(element.tags)
 
+    override fun getHighlightedElements(element: Element, getMapData: () -> MapDataWithGeometry) =
+        getMapData().filter("nodes, ways, relations with " + isKindOfShopExpression())
```

Comment:
Isn't this confusing when e.g. the name quest is open for a cemetery and all nearby shops are highlighted? (same for the opening hours quest)

Reply:
That would be shops that are as close as 30m from the center of the cemetary though. I thought about it. The name quest basically contains shops + larger things like parks, cemeteries etc.
For shops (etc) we want to have this in any case, for the larger things there is not really any need, but I think it will also not confuse because in most cases, there is no shop or anything anyway that close by.

Reply:
We **could** do something like `return if(element "is a kind of shop") mapData."elements that are a kind of shop" else null` but not sure if it is worth it